iOS/DuckDuckGo/MainViewController.swift

```diff
@@ -118,6 +118,9 @@ class MainViewController: UIViewController {
     @UserDefaultsWrapper(key: .syncDidShowSyncPausedByFeatureFlagAlert, defaultValue: false)
     private var syncDidShowSyncPausedByFeatureFlagAlert: Bool
 
+    @UserDefaultsWrapper(key: .networkProtectionEntitlementsExpired, defaultValue: false)
```

Comment:
This is interesting because I don't think we do this in macOS either (we probably should): the default should likely be `true` here as users don't have entitlements on first launch.

Having this as false by default will fire expiration pixels on first launch.

Reply:
- making @UserDefaultsWrapper more generic and cross-platform is on my todo list
- It doesn't seem that `lastKnownEntitlementsExpired` is set to true by default in mac either, is that intentional or did I miss it somewhere?